code/modules/reagents/reagent_containers/cups/drinkingglass.dm

```diff
@@ -34,7 +34,7 @@
 /obj/item/reagent_containers/cup/glass/drinkingglass/on_reagent_change(datum/reagents/holder, ...)
 	. = ..()
 	if(!length(reagents.reagent_list))
-		renamedByPlayer = FALSE //so new drinks can rename the glass
+		REMOVE_TRAIT(src, TRAIT_WAS_RENAMED, REF(src)) //so new drinks can rename the glass
```

Comment:
I think this should probably have a defined source so we know what `REF(src)` we're removing? 

```suggestion
		REMOVE_TRAIT(src, TRAIT_WAS_RENAMED, PEN_LABEL) //so new drinks can rename the glass
```

Because it's not immediately clear this is intended to remove labels applied by pens

Reply:
Oh I missed the part in the PR body that talked about this